.netlify/build.sh

```diff
@@ -17,21 +17,45 @@ mv pyo3-gh-pages netlify_build
 ## Configure netlify _redirects file
 
 # Add redirect for each documented version
+set +x  # these loops get very spammy and fill the deploy log
+
 for d in netlify_build/v*; do
     version="${d/netlify_build\/v/}"
     echo "/v$version/doc/* https://docs.rs/pyo3/$version/:splat" >> netlify_build/_redirects
+    if [ $version != $PYO3_VERSION ]; then
```

Comment:
What if this run for an unpublished version, shortly before releasing it, e.g `0.20` and `PYO3_VERSION == 0.19`. Will it create links to the older files? Do have a note for example in the releasing docs that one must publish to crates.io before running the Netlify build?

Reply:
This will be slightly racy for sure, however given we run the netlify build on every merge to main, the window where this might be wrong is relatively short.

The order of things written in `Releasing.md` implies pushing to crates.io before running the GitHub release.

We could make the crates.io push part of the release process in CI if we wanted to codify this?

Reply:
> This will be slightly racy for sure, however given we run the netlify build on every merge to main, the window where this might be wrong is relatively short.

Yeah, I think this makes it moot. So this might publish a few confused links, but eventually things will stabilize. I think this is good enough, as search engines will only eventually re-index those page in any case.